packages/sql-graphql/lib/entity-to-type.js

```diff
@@ -59,7 +59,7 @@ function constructGraph (app, entity, opts, ignore) {
         return acc
       }, {})
       try {
-        meta.type = new graphql.GraphQLEnumType({ name: key, values: enumValues })
+        meta.type = new graphql.GraphQLEnumType({ name: `${entityName}_${key}`, values: enumValues })
```

Comment:
```suggestion
        meta.type = new graphql.GraphQLEnumType({ name: `${entityName}${key}`, values: enumValues })
```

Reply:
might be that you need to pass both through `camelCase`

Reply:
Done on [4b3bbf4](https://github.com/platformatic/platformatic/pull/528/commits/4b3bbf4b510b71bf1e794c25c54a091386b7102c)